www-client/luakit/luakit-2.3.1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
Please bump to EAPI=8.

Reply:
Thanks! I updated this and the EAPI as well.